runtime/runtime/tests/test_async_calls.rs

```diff
@@ -161,38 +172,45 @@ fn test_single_promise_with_callback() {
-                        assert_eq!(function_call_action.gas, GAS_1);
-                        assert_eq!(function_call_action.deposit, 0);
-                     }
-                     => [r1, r2, ref0] );
+    let [r0] = &*assert_receipts!(group, signed_transaction) else {
+        panic!("Incorrect number of produced receipts")
+    };
+    let [r1, r2, ref0] = &*assert_receipts!(group, "near_0" => r0 @ "near_1",
+    ReceiptEnum::Action(ActionReceipt{actions, ..}), {},
+    actions,
+    a0, Action::FunctionCall(function_call_action), {
+       assert_eq!(function_call_action.gas, GAS_1);
+       assert_eq!(function_call_action.deposit, 0);
+    }) else {
+        panic!("Incorrect number of produced receipts")
+    };
     let data_id;
-    assert_receipts!(group, "near_1" => r1 @ "near_2",
-                     ReceiptEnum::Action(ActionReceipt{actions, output_data_receivers, ..}), {
-                        assert_eq!(output_data_receivers.len(), 1);
-                        data_id = output_data_receivers[0].data_id;
-                     },
-                     actions,
-                     a0, Action::FunctionCall(function_call_action), {
-                        assert_eq!(function_call_action.gas, GAS_2);
-                        assert_eq!(function_call_action.deposit, 0);
-                     }
-                     => [ref1]);
-    assert_receipts!(group, "near_1" => r2 @ "near_3",
-                     ReceiptEnum::Action(ActionReceipt{actions, input_data_ids, ..}), {
-                        assert_eq!(input_data_ids.len(), 1);
-                        assert_eq!(data_id, input_data_ids[0].clone());
-                     },
-                     actions,
-                     a0, Action::FunctionCall(function_call_action), {
-                        assert_eq!(function_call_action.gas, GAS_2);
-                        assert_eq!(function_call_action.deposit, 0);
-                     }
-                     => [ref2]);
+    let [ref1] = &*assert_receipts!(group, "near_1" => r1 @ "near_2",
+    ReceiptEnum::Action(ActionReceipt{actions, output_data_receivers, ..}), {
+       assert_eq!(output_data_receivers.len(), 1);
+       data_id = output_data_receivers[0].data_id;
+    },
+    actions,
+    a0, Action::FunctionCall(function_call_action), {
+       assert_eq!(function_call_action.gas, GAS_2);
+       assert_eq!(function_call_action.deposit, 0);
+    }
+    ) else {
+        panic!("Incorrect number of produced receipts")
+    };
+    let [ref2] = &*assert_receipts!(group, "near_1" => r2 @ "near_3",
+    ReceiptEnum::Action(ActionReceipt{actions, input_data_ids, ..}), {
+       assert_eq!(input_data_ids.len(), 1);
+       assert_eq!(data_id, input_data_ids[0].clone());
+    },
+    actions,
+    a0, Action::FunctionCall(function_call_action), {
+       assert_eq!(function_call_action.gas, GAS_2);
+       assert_eq!(function_call_action.deposit, 0);
+    }
```

Comment:
It is unfortunate that rustfmt gives up easily when formatting macros, but I believe this should be idented at least once more.

Reply:
Yeah but when I do format it correctly, rustfmt puts it back in most places :(

I managed to get it formatter a bit better in this and a few other places. But in general, the macro calls refuse to stay formatted after a rustfmt.